llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -960,6 +960,21 @@ def IREELinalgExt_WinogradInputTransformOp : IREELinalgExt_Op<"winograd.input_tr
     SmallVector<int64_t> imageDimensions() {
       return llvm::to_vector(getImageDimensions());
     }
+    SmallVector<int64_t> nhwcImageDimensions() {
+      return SmallVector<int64_t>{1, 2};
```

Comment:
We don't need to compose a new SmallVector each time given this is just constant? Consider using `std::array` and `ArrayRef`?

Reply:
Thanks will change to use that instead.